src/test/java/ReadingAggregatorTest.java

```diff
@@ -0,0 +1,125 @@
+import eventprocessing.fileservices.CSVFileService;
+import eventprocessing.models.Bucket;
+import eventprocessing.models.ReadingAggregator;
+import eventprocessing.models.SqsResponse;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.threeten.extra.MutableClock;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.temporal.ChronoUnit;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+public class ReadingAggregatorTest {
```

Comment:
Good work on making this more readable. 👍 I've left a few suggestions, but the structure of the tests is very nice.

Have you had a chance to think about injecting the config as a dependency so the tests don't make assumptions about aggregating events every minute?